Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -779,6 +792,132 @@ private void chooseController()
+    {
+      @Override public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser)
+      {
+        mTextSliderValueWidth.setText(String.valueOf(progress));
+      }
+
+      @Override public void onStartTrackingTouch(SeekBar seekBar)
+      {
+        // Do nothing
+      }
+
+      @Override public void onStopTrackingTouch(SeekBar seekBar)
+      {
+        // Do nothing
+      }
+    });
+
+
+    int irCenter = Integer.valueOf(
+            mPreferences.getString(SettingsFile.KEY_WIIBIND_IR_CENTER + mSelectedGameId, "50"));
+
+    TextView mTextSliderValueCenter = (TextView) view.findViewById(R.id.text_ir_center);
+    TextView unitsCenter = (TextView) view.findViewById(R.id.text_ir_center_units);
+    SeekBar seekbarCenter = view.findViewById(R.id.seekbar_center);
+
+    mTextSliderValueCenter.setText(String.valueOf(irCenter));
+    unitsCenter.setText(getString(R.string.center));
+    seekbarCenter.setMax(100);
+    seekbarCenter.setProgress(irCenter);
+    seekbarCenter.setKeyProgressIncrement(5);
+    seekbarCenter.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener()
+    {
+      @Override public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser)
+      {
+        mTextSliderValueCenter.setText(String.valueOf(progress));
+      }
+
+      @Override public void onStartTrackingTouch(SeekBar seekBar)
+      {
+        // Do nothing
+      }
+
+      @Override public void onStopTrackingTouch(SeekBar seekBar)
+      {
+        // Do nothing
+      }
+    });
+
+    AlertDialog.Builder builder = new AlertDialog.Builder(this);
+    builder.setTitle(getString(R.string.emulation_ir_sensitivity));
+    builder.setView(view);
+    builder.setPositiveButton(R.string.ok, (dialogInterface, i) ->
+    {
+      SettingsFile.saveSingleCustomSetting(mSelectedGameId, Settings.SECTION_CONTROLS,
+              SettingsFile.KEY_WIIBIND_IR_HEIGHT, mTextSliderValueHeight.getText().toString());
+      SettingsFile.saveSingleCustomSetting(mSelectedGameId, Settings.SECTION_CONTROLS,
+              SettingsFile.KEY_WIIBIND_IR_WIDTH, mTextSliderValueWidth.getText().toString());
+      SettingsFile.saveSingleCustomSetting(mSelectedGameId, Settings.SECTION_CONTROLS,
+              SettingsFile.KEY_WIIBIND_IR_CENTER, mTextSliderValueCenter.getText().toString());
+
+      NativeLibrary.ReloadWiimoteConfig();
+
+      SharedPreferences.Editor editor = mPreferences.edit();
+      editor.putString(SettingsFile.KEY_WIIBIND_IR_HEIGHT + mSelectedGameId,
+              mTextSliderValueHeight.getText().toString());
+      editor.putString(SettingsFile.KEY_WIIBIND_IR_WIDTH + mSelectedGameId,
+              mTextSliderValueWidth.getText().toString());
+      editor.putString(SettingsFile.KEY_WIIBIND_IR_CENTER + mSelectedGameId,
+              mTextSliderValueCenter.getText().toString());
+      editor.apply();
+    });
+    builder.setNegativeButton(R.string.cancel, (dialogInterface, i) ->
+    {
+
```

Comment:
Intentionally left blank?

Reply:
Yeah, I'll throw a 'Do Nothing' comment in there.